src/status_im/utils/universal_links/core.cljs

```diff
@@ -105,7 +114,11 @@
     (handle-public-chat cofx (match-url url public-chat-regex))
 
     (spec/valid? :global/public-key (match-url url profile-regex))
```

Comment:
In QR reader there is a different check which uses etherium resolver, but here and in other places it is just regexp. Not sure if it should be unified somehow

Reply:
This is the root of the problem described here https://github.com/status-im/status-react/pull/9811#discussion_r367270363

Reply:
Not sure if it helps, but there are different versions of QR code reader. QR code reader in Wallet checks for accurate Ethereum account address for transactions. QR code reader in Chat checks for valid chat key (public key) or username.